.github/workflows/ci.yaml

```diff
@@ -17,8 +17,8 @@ jobs:
         - '2.7'
         - '3.0'
         vddk-version:
-        - '5.5.0'
         - '6.7.0'
+        - '8.0.0'
```

Comment:
Should we also include 7.x here?

Reply:
There is a test that checks for ConnectParams prior to 6.5.0 so if we're going to drop 5.5.0 we should add 6.0.0 (https://github.com/ManageIQ/ffi-vix_disk_lib/commit/2b4dacba353b1dfa37775e0e342b3e43314b4aaa#diff-f114eae0a13bc4bbd685a6a1dc8432e260f3a3986ac42a6b27d177f29dd1a911R26-R35)

Reply:
If there are differences in the actual API though, then we need to update the fake library.  Since we can't redistribute the VDDK, instead we build a fake library (see https://github.com/ManageIQ/ffi-vix_disk_lib/tree/master/spec/ext) and the VDDK_VERSION drives the build of that.  If there are differences in APIs, we probably should update the fake library to produce the expected API.

Reply:
Sure we can throw some `#ifdef`'s in there, but the test is actually for our `lib/ffi-vix_disk_lib/struct.rb` code since we have to have the definition of the `ConnectParams` struct there for ffi

Reply:
Ah ok, that's fair - when I saw `ConnectParams` comment I thought that was part of the VDDK libraries API itself as opposed to our conditionals in front of it.